mayastor/src/bdev/nexus/nexus_bdev.rs

```diff
@@ -481,21 +488,26 @@ impl Nexus {
     }
 
     pub async fn sync_labels(&mut self) -> Result<(), Error> {
-        let label = self.update_child_labels().await.context(WriteLabel {
+        if env::var("NEXUS_DONT_READ_LABELS").is_ok() {
```

Comment:
I wonder if there is a better way to do this?

Reply:
As per the comments, this really only exists for one particular use case - where we want to use NULL bdevs purely for the purpose of performance testing, and as such is only ever needed in testing. I don't really want to see this exposed elsewhere, but am always open to suggestions about how to do it better. :)

Reply:
The existing `NEXUS_LABEL_IGNORE_ERRORS` environment variable is also used in the `nvmet` binary. It should either be kept in sync or the environment variable name changed.